python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -218,6 +220,44 @@ def get_metadata(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, Any
         """
         return default_metadata_from_dbt_resource_props(dbt_resource_props)
 
+    @classmethod
+    @public
+    def get_tags(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, str]:
+        """A function that takes a dictionary representing properties of a dbt resource, and
+        returns the Dagster tags for that resource.
+
+        Note that a dbt resource is unrelated to Dagster's resource concept, and simply represents
+        a model, seed, snapshot or source in a given dbt project. You can learn more about dbt
+        resources and the properties available in this dictionary here:
+        https://docs.getdbt.com/reference/artifacts/manifest-json#resource-details
+
+        This method can be overridden to provide custom tags for a dbt resource.
+
+        Args:
+            dbt_resource_props (Mapping[str, Any]): A dictionary representing the dbt resource.
+
+        Returns:
+            Mapping[str, str]: A dictionary representing the Dagster tags for the dbt resource.
+
+        Examples:
+            .. code-block:: python
+
+                from typing import Any, Mapping
+
+                from dagster_dbt import DagsterDbtTranslator
+
+
+                class CustomDagsterDbtTranslator(DagsterDbtTranslator):
+                    @classmethod
+                    def get_tags(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, str]:
+                        return {"custom": "tag"}
+        """
+        if dbt_resource_props["resource_type"] == "model":
+            tags = dbt_resource_props.get("tags", [])
+            return {tag: TAG_NO_VALUE for tag in tags if is_valid_definition_tag_key(tag)}
+        else:
+            return {}
```

Comment:
We don't need the conditional. Otherwise, we won't apply this to dbt seeds.

```suggestion
        return {tag: TAG_NO_VALUE for tag in dbt_resource_props.get("tags", []) if is_valid_definition_tag_key(tag)}
```